### PR TITLE
extend UTF-8 support for issue #723

### PR DESCRIPTION
**Pull request by davidcarlisle**
Extend UTF-8 support as requested in issue #723

# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [X] Test file(s) added
- [X] Version and date string updated in changed source files
- [X] Relevant `\changes` entries in source included
- [X] Relevant `changes.txt` updated
- [X] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/utf8ienc.dtx [hunk 1]
```diff
@@ -1368,6 +1368,18 @@
 %<all,t1,ly1>\DeclareUnicodeCharacter{017E}{\v z}
 %<all,ts1,ly1>\DeclareUnicodeCharacter{0192}{\textflorin}
 %    \end{macrocode}
+% \changes{v1.2n}{2021/12/10}{add 01C4-01CC}
+%    \begin{macrocode}
+%<all,t1>\DeclareUnicodeCharacter{01C4}{D\v Z}
+%<all,t1>\DeclareUnicodeCharacter{01C5}{D\v z}
+%<all,t1>\DeclareUnicodeCharacter{01C6}{d\v z}
```
Comment (FrankMittelbach) [inline base/utf8ienc.dtx:1375]: I kow that are are still people using LY1 and all such definitions would work for them too wouldn't they? so perhaps we should always set all,T1,LY1 if that is possible. (might also apply to other chars we added recently).

**Review by FrankMittelbach — APPROVED** [review]
one comment for consideration

File: base/utf8ienc.dtx (hunk 1 above)
Comment (davidcarlisle) [reply base/utf8ienc.dtx:1375]: @FrankMittelbach  and OT1 but actually it doesn't make any difference these days as we pre-load OT1 and T1 into the format so any UTF-8 declaration in T1 is available whatever encodings you use.

Comment (FrankMittelbach) [reply base/utf8ienc.dtx:1375]: true, so forget my remark (which you probably already did :-))

**Pull request merged**